layers/meta-resin-intel/recipes-core/initrdscripts/files/init-external-media.sh

```diff
@@ -43,14 +43,26 @@ read_args() {
     done
 }
 
-boot_external_rootfs() {
-    mkdir /.flasher_root
-    mount /dev/disk/by-label/flash-root /.flasher_root
+boot_rootfs() {
+   local _what=$1
+
+    echo "init-external-media.sh: Waiting for udev to populate /dev/disk/by-label/$_what... "
+    while true
+    do
+        if ls -A /dev/disk/by-label/$_what >/dev/null 2>&1; then
```

Comment:
$_what is a filename. Filenames can contain any character other than "/" and "\0", including a space. By not quoting it here we lose that. While unlikely it can lead to bugs.